config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/config/loader/JmsDestinationLoaderTest.java

```diff
@@ -83,7 +83,7 @@ void testLoad_GenericInboundYaml() throws IOException {
                 "    acknowledgeType: \"ON_COMPLETION\"\n" +
                 "    replyType: \"SPECIFIED_QUEUE\"\n" +
                 "    replyToQueueName: \"my-reply-Q\"\n" +
-                "    useRequestCorrelationId: false\n" +
+                "    getUseRequestCorrelationId: false\n" +
```

Comment:
This might be a mistake. I believe these still should be the field name (not the getter method name).